src/Ledger/Ratify.lagda

```diff
@@ -263,39 +266,83 @@ actualVotes Γ pparams cc ga votes
   roleVotes r = mapKeys (uncurry credVoter) (filter (λ (x , _) → r ≡ proj₁ x) votes)
 
   activeDReps = dom (filter (λ (_ , e) → currentEpoch ≤ e) dreps)
-  spos = filterˢ IsSPO (dom (stakeDistr stakeDistrs))
+  spos  = filterˢ IsSPO (dom (stakeDistr stakeDistrs))
+\end{code}
+\begin{code}
 
   getCCHotCred : Credential × Epoch → Maybe Credential
   getCCHotCred (c , e) = case ¿ currentEpoch ≤ e ¿ᵇ , lookupᵐ? ccHotKeys c of
 \end{code}
 \begin{code}[hide]
-    λ where
+      λ where
+\end{code}
+\begin{code}
+        (true , just (just c'))  → just c'
+        _                        → nothing -- expired, no hot key or resigned
+\end{code}
+\begin{code}[hide]
+  getPoolParams : Credential → Maybe PoolParams
+  getPoolParams (KeyHashObj kh) = lookupᵐ? pools kh
+  getPoolParams _ = nothing
+\end{code}
+\begin{code}
+
+  SPODefaultVote : GovAction → VDeleg → Vote
+  SPODefaultVote (TriggerHF _) _ = Vote.no
+  SPODefaultVote NoConfidence (credVoter SPO c) = case getPoolParams c of
+\end{code}
+\begin{code}[hide]
+      λ where
+\end{code}
+\begin{code}
+        nothing → Vote.no
```

Comment:
I tried that but it breaks the alignment (probably has to do with the fact that the thing we're trying to align with, `λ where`, is hidden).  Specifically, the `nothing → Vote.no` ends up on the far left of the page, as if it's supposed to be outside the proof term block.

Reply:
You have two spaces below between `just p`, maybe that's the reason is breaks

Reply:
Actually, I put the extra space there on purpose so that the lines below the `λ where` would have something to align with (the `p`).  Otherwise, they get left-justified, for some reason. 😕

Reply:
![misalign-left](https://github.com/user-attachments/assets/685d0369-f1de-4ca8-a91e-5c6154dfa31e)